server/src/services/jwt.service.ts

```diff
@@ -0,0 +1,28 @@
+import jwt from 'jsonwebtoken';
+// import UserRepository from '../repositories/user.repository';
+import env from '../config';
```

Comment:
```suggestion
import config from '../config';
```

다른 데서는 config 로 가져오셨길래? ㅋㅋ

Reply:
아 이겈ㅋㅋㅋㅋㅋ 균님꺼랑 맞추다게 제대로 수정안됬네요...